subsys/bluetooth/Kconfig.scan

```
@@ -5,7 +5,7 @@
 #
 
 menuconfig BT_SCAN
-	depends on BT_CENTRAL
+	depends on BT_CENTRAL || BT_OBSERVER
```

BT_CENTRAL depends on BT_OBSERVER so this modification does not change anything. I guess we should simply use the following dependency:

```suggestion
	depends on BT_OBSERVER
```

as you can use this module with BT_CENTRAL option disabled

---

Good point.